.github/workflows/slovensko_digital_ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Slovensko.Digital CI
+
+on:
+  push:
+    branches: '**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      PGHOST: localhost
+      RAILS_ENV: test
+
+    services:
+      postgres:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_USER: runner
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - run: yarn install
+      - run: bundle exec rails db:create db:structure:load --trace
+
+      - name: Setup Code Climate test-reporter
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./test-reporter
+          chmod +x ./test-reporter
+          ./test-reporter before-build
+      - name: Build and test with RSpec
```

Comment:
tieto `name` to podla mna len zneprehladnuju, netreba to

Reply:
Ja by som ich tu nechal.